Oqtane.Server/Infrastructure/LogManager.cs

```diff
@@ -207,7 +207,7 @@ private void SendNotification(Log log)
                 {
                     if (userrole.Role.Name == RoleNames.Host)
                     {
-                        var url = _accessor.HttpContext.Request.Scheme + "://" + _tenantManager.GetAlias().Name + "/admin/log";
+                        var url = $"{_accessor.HttpContext.Request.Scheme}://{_tenantManager.GetAlias().Name}/admin/log?id={log.LogId}";
```

Comment:
If you specifiy the id here how the log will show all if the id not specified?

Reply:
why? you want to see concrete item from notification email.

Reply:
Oops, I didn't notice that the change inside `SendNotifications`